html-forms_formvalidation/index.html

```diff
@@ -32,21 +32,26 @@ <h2>Customer complaint form</h2>
             name="firstName"
             id="first-name"
             placeholder="Your first name"
+            minlength="4" 
+            maxlength="30" required
           />
 
+
           <label for="last-name">Last Name</label>
           <input
             type="text"
             name="lastName"
             id="last-name"
             placeholder="Your last name"
+            minlength="4" maxlength="30" required
           />
 
           <label for="age">Age</label>
-          <input type="number" name="age" id="age" />
+          <input type="number" name="age" id="age" min="14" required />
 
           <label for="email">Email</label>
-          <input type="text" name="email" id="email" />
+          <input type="text" name="email" id="email" required />
```

Comment:
Hear must be type="email" Otherwise there will be no validation for the @ symbol

Reply:
Nice! Danke!